packages/perseus/src/__stories__/server-item-renderer.stories.tsx

```diff
@@ -42,3 +44,37 @@ export const WithLintingError = (args: StoryArgs): React.ReactElement => {
         />
     );
 };
```

Comment:
I added two new stories to the Server Item Renderer as I felt the current examples were far too basic to test with:

1.  InputNumberWithInteractionCallback 
This story renders two input number widgets and console logs the response data. It's helpful for testing simple multi-widget states and scoring. 

2. MultiWidgetWithInteractionCallback
This story renders two different widgets (expression and radio) with the interaction callback, so that we can test multi-widget states, scoring, and the interactionCallback. 

I kept the console log in the interactionCallback as I felt like it provides decent value for testing, but I'd be happy to remove it if we feel it isn't appropriate to override eslint to console log in a Storybook. If it is felt that these new stories are unhelpful, I'd also be happy to simply remove them entirely.

Reply:
> I kept the console log in the interactionCallback as I felt like it provides decent value for testing

I personally don't mind console logs, but I think you could do this with Storybook Actions and the data would show up in the SB UI.